tests/end_to_end/netsh_test_helper.h

```diff
@@ -20,14 +20,14 @@
 
 DWORD WINAPI
 PreprocessCommand(
-    _In_opt_ HANDLE hModule,
+    _In_opt_ const HANDLE hModule,
     _Inout_updates_(dwArgCount) LPWSTR* ppwcArguments,
-    _In_ DWORD dwCurrentIndex,
-    _In_ DWORD dwArgCount,
+    _In_ const DWORD dwCurrentIndex,
+    _In_ const DWORD dwArgCount,
```

Comment:
don't use const with value arguments, only pointers.  I think we only have `_In_` here because the netsh prototypes incorrect have them.